### ISSUE TITLE
Your title should make sense if said after "The issue is <your issue title>"

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#1487]
## Description of the issue

"Publication" tab in scenario bundles displays date where it should display link to study report (see screenshot)
![grafik](https://github.com/OpenEnergyPlatform/oeplatform/assets/36844029/debca99f-4d99-4e1e-9e5a-4ed1a30caa68)

It would be great if this could be fixed quickly. 

## Steps to Reproduce
1. navigate to any publication tab of a scenario bundle 
2.
3.

FY @wingechr

**Comment by adelmemariani** [comment]
Yes, that's right and it is a bug in rendering the page. The data is stored and retrieved but rendered wrongly. 

[Here](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/factsheet/frontend/src/components/factsheet.js#L1962) is the bug.

The data is loaded in the background as below:
![Screenshot 2023-12-04 at 14 53 45](https://github.com/OpenEnergyPlatform/oeplatform/assets/7637364/75b07b2f-62bb-4790-9b3f-fa0993eb94ec)

**Comment by jh-RLI** [comment]
We can do a hotfix if you have time this week @adelmemariani

**Comment by jh-RLI** [comment]
Great, can you open up issues in the oeplatform repo here? so we can reference them :)

**Comment by adelmemariani** [comment]
Yes. I guess @stap-m will create the issues.

**Comment by stap-m** [comment]
see #1488 and #1489

**Comment by jh-RLI** [comment]
Thanks :)
